<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tampil</class>
 <widget class="QMainWindow" name="Tampil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Server</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QPushButton" name="PB_kirim">
      <property name="text">
       <string>Kirim</string>
      </property>
     </widget>
    </item>
    <item row="19" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Haliza</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="TreeSubwin" name="treeView"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QPushButton" name="PB_compare">
      <property name="text">
       <string>Compare</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Sofhaliza</string>
      </property>
     </widget>
    </item>
    <item row="19" column="0" colspan="2">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="PB_connect">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="13" column="0">
     <widget class="QPushButton" name="PB_synchron">
      <property name="text">
       <string>Synchronize</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeSubwin</class>
   <extends>QTreeView</extends>
   <header>treesubwin.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
